pubspec.yaml

```diff
@@ -216,6 +213,11 @@ dependencies:
 
   focused_menu_custom: 1.2.0
 
+  flutter_typeahead:
+    git:
+      url: https://github.com/dab246/flutter_typeahead.git
```

Comment:
what happen with it?

Reply:
Well, I wanted to add support for the `autofillhints` property of `TextField` so I created a new branch to handle it.

Reply:
please contribute to upstream :)

Reply:
Very difficult, currently their repository is supporting flutter `3.7.x` so we need to wait for `Tmail` to be upgraded to fluttter version `3.7.7.5`